assets/css/default-calendar-grid.css

```diff
@@ -559,7 +559,12 @@
 	.simcal-print-calendar .print-calendar-button,
 	.print-calendar-button,
 	.simcal-print-calendar .simcal-nav .simcal-prev,
-	.simcal-print-calendar .simcal-nav .simcal-next {
+	.simcal-print-calendar .simcal-nav .simcal-next,
```

Comment:
Why are we changing CSS of grid when we want this change for fullcalendar?

Reply:
this css is apply on the print screen and we have common css for grid list and fullcalendar grid view.